.github/workflows/sanity-test.yaml

```diff
@@ -0,0 +1,92 @@
+name: Sanity tests
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - stable-*
+    tags:
+      - '*'
+
+jobs:
+  sanity:
+    uses: ansible-network/github_actions/.github/workflows/sanity.yml@main
+    with:
+      matrix_include: "[]"
+      matrix_exclude: >-
```

Comment:
I feel like we should be able to update the ansible-network workflow to only test 2.14+ at this point, which should remove the need for us to maintain any kind of excludes here.